content/en/docs/demos/ingress_traefik.md

```diff
@@ -0,0 +1,162 @@
+
+## Prerequisites 
+
+* Kubernetes cluster version v1.19.0 or higher.
+* Use kubectl to interact with the API server.
+* osm-edge is not installed, and must be removed first if installed.
+* osm cli is installed to install osm-edge.
+* Helm 3 command line tool is installed for traefik installation.
+* osm-edge version >= v1.1.0.
+
+## Demo
+
+### Install Traefik
+
+```shell
+helm repo add traefik https://helm.traefik.io/traefik
+helm repo update
+helm install traefik traefik/traefik -n traefik --create-namespace
+```
+
+Verify that the pod is up and running.
+
+```shell
+kubectl get po -n traefik
+NAME READY STATUS RESTARTS AGE
+traefik-69fb598d54-9v9vf 1/1 Running 0 24s
+```
+
+Retrieve and store external IP address and port of the entry gateway to environment variables, which will be used later to access the application.
+
+```
+export ingress_host="$(kubectl -n traefik get service traefik -o jsonpath='{.status.loadBalancer.ingress[0].ip}')"
+export ingress_port="$(kubectl -n traefik get service traefik -o jsonpath='{.spec.ports[? (@.name=="web")].port}')"
+```
+
+### Install osm
+
+```shell
+export osm_namespace=osm-system 
+export osm_mesh_name=osm 
+
+osm install \
+    --mesh-name "$osm_mesh_name" \
+    --osm-namespace "$osm_namespace" \
+    --set=osm.enablePermissiveTrafficPolicy=true
+```
+
+Confirm that the pod is up and running.
+
+```shell
+kubectl get po -n osm-system
+NAME READY STATUS RESTARTS AGE
+osm-bootstrap-6477f776cc-d5r89 1/1 Running 0 2m51s
+osm-injector-5696694cf6-7kvpt 1/1 Running 0 2m51s
+osm-controller-86d68c557b-tvgtm 2/2 Running 0 2m51s
+```
+
+### Deploy sample service
+
+```shell
+kubectl create ns httpbin
+osm namespace add httpbin
+kubectl apply -f https://raw.githubusercontent.com/flomesh-io/osm-edge-docs/main/manifests/samples/httpbin/httpbin.yaml -n httpbin
+```
+
+Confirm that the service has been created and the pod is up and running.
+
+```shell
+kubectl get svc -n httpbin
+NAME TYPE CLUSTER-IP EXTERNAL-IP PORT(S) AGE
+httpbin ClusterIP 10.43.51.114 <none> 14001/TCP 9s
+
+kubectl get po -n httpbin
+NAME READY STATUS RESTARTS AGE
+httpbin-69dc7d545c-bsjxx 2/2 Running 0 77s
+```
+
+### HTTP Ingress
+
+Next, create an ingress to expose the `14001` port of the `httpbin` service under the `httpbin` namespace.
+
+```shell
+kubectl apply -f - <<EOF
+kind: Ingress
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: httpbin
+  namespace: httpbin
+  annotations:
+    kubernetes.io/ingress.class: "traefik"
+spec:
+  rules:
+  - host: httpbin.org
+     http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: httpbin
+            port:
+              number: 14001
+EOF
+```
+
+Using the entry gateway address and port saved earlier to access the service, you should receive a response of `502` at this point. This is normal, because you still need to create ``IngressBackend`` to allow the entry gateway to access the ``httpbin`` service.
```

Comment:
```suggestion
Using the entry gateway address and port saved earlier to access the service, you should receive a response of `502` at this point. This is normal, because you still need to create `IngressBackend` to allow the entry gateway to access the `httpbin` service.
```